discord/asset.py

```diff
@@ -0,0 +1,27 @@
+import io
+from typing import Union, TYPE_CHECKING
+
+from pydantic import BaseModel
+
+
+if TYPE_CHECKING:
+    from discord import Client
+
+
+CDN_URL = "https://cdn.discordapp.com"
```

Comment:
how is the CDN_URL used here? It has no reference currently.

Reply:
Will remove it